### ISSUE TITLE
Replace the Hard Disk drives on all the "Bubblino" PC's with SSD's

### ISSUE THREAD

**Issue by magman2112** [DoESLiverpool/somebody-should#1102]
I have managed to find some 120GB SSD drives at very reasonable prices, so I have ordered 3 so that all three PC's that are used to manage equipment, for the laser cutters and dye Sublimation printers, can be sped up considerably by replacing their hard drives.

We should be able to clone the existing hard drives, so there should hopefully be no need to re-install any software, but it may also be worth investigating licensing issues at the same time.

- [x] Dye Sublimation PC Upgrade
- [ ] Sophia PC Upgrade
- [ ] Gerald PC Upgrade

**Comment by magman2112** [comment]
The new SSD Drives and some holders (to fit in a 3.5" disk space if required) are now in DoES. We just need to find some free time on the machines to mirror the existing drives and install them.

**Comment by MatthewCroughan** [comment]
@magman2112 I noticed you replaced the drive in the vinyl cutting PC yesterday. That system is now not booting, did you test it working before you were done? I need to know where the hard drive is so I can replace it for the time being.

**Comment by magman2112** [comment]
I actually put the original drive back in yesterday, not the SSD. it appeared to be booting when I looked at it, it was requesting a password anyway, but I got distracted with something else and forgot to check on it before I left.

**Comment by amcewen** [comment]
I turned the computer off and on, and it's booted up fine.  @zarino is using it to print labels as I type...

**Comment by MatthewCroughan** [comment]
@magman2112 Right, for some reason 1 in a million, it will stall on boot, maybe the drive is actually dying at this stage. Would you like me to carry on with this?

**Comment by magman2112** [comment]
The drive on the Sublimation PC has now been upgraded. I will keep hold of the old drive for now so that we can revert if necessary (it is in my box on the members shelves).

**Comment by MatthewCroughan** [comment]
@magman2112 I'm closing this since it seems to have been a success.

**Comment by magman2112** [comment]
Nope, not yet. There are still 2 PC’s that need to be upgraded.

**Comment by DoESsean** [comment]
@magman2112 Did the other two PCs get upgraded? Are the SSDs around somewhere so that someone else might be able to do it if they have chance?

**Comment by MatthewCroughan** [comment]
@DoESsean @magman2112 If I'm told were the SSDs are, I can do this today, tomorrow, etc.

**Comment by magman2112** [comment]
@MatthewCroughan , these are in my box on the shelves by the electronics bench. There should also be a USB - SATA controller in there as well.

**Comment by MatthewCroughan** [comment]
@magman2112 Thanks for getting back so quick, I'll get to it.
